src/library.js

```diff
@@ -128,7 +128,11 @@ addToLibrary({
     if (keepRuntimeAlive() && !implicit) {
       var msg = `program exited (with status: ${status}), but keepRuntimeAlive() is set (counter=${runtimeKeepaliveCounter}) due to an async operation, so halting execution but not exiting the runtime or preventing further async execution (you can use emscripten_force_exit, if you want to force a true shutdown)`;
 #if MODULARIZE
-      readyPromiseReject(msg);
+      if (!runtimeInitialized) {
+        // If the runtime has not yet been initializated then reject the
+        // ready promise instead of logging the error here.
+        readyPromiseReject(msg);
+      } else
```

Comment:
This is during the shutdown code. Why do we need to check if the runtime was initialized right after logging that we exited?

Reply:
This is no longer part of this PR